cmake/linux/generate_icons_linux.sh.in

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Recolor icon
+echo Recoloring vector icon...
+case "$1" in
+	"-r" ) # Release
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	"-b" | "-rc" ) # Beta/Release Candidate
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	"-c" | "-n" ) # Canary/Nightly
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' ../branding/icon.svg > recolored-icon.svg
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' ../branding/splash.svg > recolored-splash.svg
+	;;
+	* ) # No type specified, leave it be
+	sed '' ../branding/icon.svg > recolored-icon.svg
+	sed '' ../branding/splash.svg > recolored-splash.svg
+	;;
+esac
+
+# Generate raster images
+echo Generating raster images for icon...
+for RES in 16 24 32 48 64 96 128
```

Comment:
This technique is great, but why isn't it used in all scripts?  Some use multiple separate calls, others use `for` loops.

Reply:
The ones that don't use `for` loops started out as a copy-paste from the example you shared in #3347. I've updated all scripts to use `for` loops. Fixed 👍